fastlane/Fastfile

```diff
@@ -358,8 +358,15 @@ platform :ios do
     replace_assets
 
     sh 'cd ../ios && xcodebuild -workspace Mattermost.xcworkspace -scheme Mattermost -arch x86_64 -sdk iphonesimulator -configuration Release -parallelizeTargets -derivedDataPath ../build-ios ENABLE_BITCODE=NO CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO ENABLE_BITCODE=NO'
-    sh 'cd ../build-ios/Build/Products/Release-iphonesimulator && zip -r Mattermost-simulator-x86_64.app.zip Mattermost.app'
-    sh 'mv ../build-ios/Build/Products/Release-iphonesimulator/Mattermost-simulator-x86_64.app.zip ../'
+
+    data_path = 'ios'
+    if ENV['CIRCLECI'] == 'true'
+      data_path = 'build-ios'
+    end
+
+    sh "cd ../#{data_path}/Build/Products/Release-iphonesimulator && zip -r Mattermost-simulator-x86_64.app.zip Mattermost.app"
+    sh "mv ../#{data_path}/Build/Products/Release-iphonesimulator/Mattermost-simulator-x86_64.app.zip ../"
```

Comment:
I'll submit this change later separately on upstream but will leave as is here temporarily for review process (in case someone wants to checkout and try to run locally).